package/yast2-bootloader.changes

```diff
@@ -1,10 +1,18 @@
 -------------------------------------------------------------------
+Mon May  2 10:24:17 CEST 2016 - schubi@suse.de
+
+- Fixed nil in custom_devices. [[]].compact! is returning nil.
```

Comment:
I thought the array contained Strings (or nil), so the `[[]]` is misleading, isn't it?